PORTAL/jobs/_pyperformance.py

```diff
@@ -825,7 +829,7 @@ class PyperfComparison(_PyperfComparison):
                          'bench baseline baseresult source result comparison')
 
     @classmethod
-    def _parse_value(cls, valuestr: str):
+    def _parse_value(cls, valuestr: str) -> Any:
```

Comment:
It works, but I have to `assert` that the result of `PyperfComparisonValue.parse` isn't `None` (since it can't grok that `fail = True` makes that impossible).